foreman/requirements.txt

```diff
@@ -7,11 +7,16 @@
 amqp==2.1.4               # via kombu
 billiard==3.5.0.2         # via celery
 celery==4.0.2
-django==1.10.6
+django==1.11
```

Comment:
I see you are running different versions of django in different parts of this project. I'm just curious, any reason for that?